docs/components/ButtonView.jsx

```diff
@@ -111,21 +102,28 @@ export default function ButtonView() {
             }}
             size={Button.Size.S}
             type="link"
-            value={
-              <span>
-                <code>focus()</code> ...but just for a second
-              </span>
-            }
-          />
+          >
+            <span>
+              <code>focus()</code> ...but just for a second
+            </span>
+          </Button>
         </ExampleCode>
       </Example>
 
       <PropDocumentation
         availableProps={[
+          {
+            name: "children",
+            type: "ReactNode",
+            optional: true,
+            description:
+              "The content that appears in the button, and takes precedence over 'value'",
+          },
           {
             name: "value",
-            type: "string",
-            description: "The text that appears on the button",
+            type: "ReactNode",
+            optional: true,
+            description: "The content that appears in the button if children is not passed in",
           },
```

Comment:
> Should we change all examples to use children instead of value? If we really wanted to be consistent everywhere and encourage using children, we could also make the same change in all components that contain Buttons.

Nah, I'd say don't worry about it! The reality is that, even if we were to update all examples in clever-components, people would still see examples of the old `value` prop in code, which they're likely to copy.

The one other thing that I might do to make it clear that `children` is the way of the future is to add a deprecation note to the description of the value prop, e.g.

```
"Deprecated. Use children instead. This content will appear in the button if children are not specified."
```